src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,29 @@
+import './index.scss';
+
+type color = 'primary' | 'blue' | 'green' | 'purple' | 'orange';
+
+interface TagProps {
+  color?: color;
+  style?: React.CSSProperties;
+  children: React.ReactNode;
+  onClick?: (event?: React.MouseEvent) => void;
+}
+
+const Tag = ({
+  children,
+  color = 'primary',
+  style,
+  onClick,
+  ...props
+}: TagProps) => {
+  return (
+    <div
+      className={`tag tag--${color} ${onClick ? 'pointer' : ''}`}
```

Comment:
`onClick`이 없을 때는 className을 추가 안해줘도 되니 `&&` 연산자를 사용해도 좋을 것 같아요 ! ㅎㅎㅎ
```suggestion
      className={`tag tag--${color} ${onClick && 'pointer'}`}
```

Reply:
<img width="317" alt="image" src="https://github.com/New-Barams/this-year-ajaja-fe/assets/61609327/8f93be86-fb70-4909-a5d9-43888ef744cd">
      `className={`tag tag--${color} ${onClick && 'pointer'}`}` 로 하면 클래스네임이 'undefined' 가 들어가는데 괜찮을 까요? 그래서 삼항연산자로 하긴했는데  그냥 빈칸도 별로인것 같긴해요

Reply:
헉 그렇군요 !! 그럼 저희 그냥 `classNames` 라이브러리 쓰는 건 어떤가여?? ㅎㅎㅎ 
<img width="541" alt="image" src="https://github.com/New-Barams/this-year-ajaja-fe/assets/31370590/87f4abc4-e7b0-4596-b36f-a5aa5879721c">

Reply:
다 이유가 있었네요 ㅎㅎ👍

Reply:
저는 classNames 쓰는게 좋을 것 같습니다